### DISCUSSION TITLE
Suggestion for the message after running the cli

### DISCUSSION THREAD

**Discussion by onurusluca** [obytes/react-native-template-obytes#238] [Ideas] [answered]
After the project is created, we see this message:
![image](https://github.com/obytes/react-native-template-obytes/assets/51825926/886c81b3-0aa9-43df-826f-07e9ca2ee1e5)

This also should say something like `cd MyApp`.

**Comment by yjose** [comment, marked as answer]
Agree , just create an issue to track It
